### PR TITLE
Added form field, updated unit test & bumped ver

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22353)

During testing of the 686c form it was discovered that we were missing a question about weather or not the user is the parent or guardian of the student that is being reported. This PR adds that question in, updates the corresponding unit test, and bumps the `vets-json-schema` version to include [the update](https://github.com/department-of-veterans-affairs/vets-json-schema/commit/ccf4b138584046eef02577bae8deca17ee1a721c) for this question.

## Testing done
Unit test updated

## Screenshots

Screen Shot 2021-03-30 at 11 14 57 AM

## Acceptance criteria
- [x] The question is in place on the proper page
- [x] A member of the backend team has been made aware for logic and implementation
- [x] A PR has been requested for merging

### CODE REVIEW

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ export const uiSchema = {
           isChapterFieldRequired(formData, 'report674'),
       },
     },
+    isParentOrGuardian: {
+      'ui:title': 'Is this your child, or are you their guardian?',
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: The wording of this question feels sort of awkward. Feels like it'd be better to ask something like `Are you either a parent or guardian to this child?`

**Review by micahchiang — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: @micahchiang or just `are you a parent or guardian of this child?` @jason-gcio what do you think?

Comment (jason-gcio) [reply src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: Nice catch

**Review by theodur — APPROVED** [review]

**Pull request merged**